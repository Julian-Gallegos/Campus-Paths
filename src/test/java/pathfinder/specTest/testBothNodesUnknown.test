# Find a path with Dijkstra's algorithm when
# the starting node and destination node do not exist

CreateGraph simpleGraph

FindPath simpleGraph A B