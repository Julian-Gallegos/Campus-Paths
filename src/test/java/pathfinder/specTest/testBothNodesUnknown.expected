# Find a path with Dijkstra's algorithm when
# the starting node and destination node do not exist

created graph simpleGraph

path from A to B:
unknown node A
unknown node B